src/pudl/transform/params/ferc1.py

```diff
@@ -4065,6 +4065,11 @@
         "unstack_balances_to_report_year_instant_xbrl": {
             "unstack_balances_to_report_year": True
         },
+        "reconcile_table_calculations": {
+            "column_to_check": "ending_balance",
+            "calculation_tolerance": 0.2,
```

Comment:
should the tolerance be this high? maybe a copy/paste from the other deprish table?

Reply:
Not at all, this is a relic of my checking oops. Will delete and leave this as the default threshold.